vendor/ember-suave/qunit-configuration.js

```diff
@@ -0,0 +1,3 @@
+/* globals QUnit */
+
+QUnit.config.urlConfig.push({ id: 'nojscs', label: 'Disable JSCS' });
```

Comment:
This needs to guard for QUnit being undefined. ember-suave should work properly (with some project level configuration tweaks) with ember-mocha.